stdlib/2/os/__init__.pyi

```diff
@@ -130,7 +133,14 @@ def setuid(uid: int) -> None: ...  # Unix only
 def strerror(code: int) -> str: ...
 def umask(mask: int) -> int: ...
 def uname() -> Tuple[str, str, str, str, str]: ...  # Unix only
-def unsetenv(varname: str) -> None: ...
+
+@overload
+def getenv(key: Text) -> Optional[str]: ...
```

Comment:
No, merging is great! It's much more maintainable that way. I'll take another look at your open PRs tonight (PST) and then hopefully merge them.